src/plugins/intel_cpu/src/emitters/tpp/x64/jit_debug_emitter.hpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2020-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
```

Comment:
As I mentioned in one of the previous comments, the `DEBUG_TPP_EMITTERS` define is deprecated.
We can re-introduce it just to avoid this class' compilation, but I'm not sure it's worth it.